client/router.js

```diff
@@ -352,14 +381,51 @@ Router.prototype._invalidateTracker = function() {
 Router.prototype._updateCallbacks = function () {
   var self = this;
 
-  // add global middleware
   self._page.callbacks = [];
+  self._page.exits = [];
+
+  // add global enter triggers
+  self._page("*", function(ctx, next) {
+    _.each(self._triggersEnter, function(fn) {
+      if (typeof fn === 'function') {
+        fn(ctx);
```

Comment:
Do you mean calling every triggers with `self._current` like `fn(self._current)`?